db/migrations/public/20231127222209_create-licences-view.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const viewName = 'licences'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      // NOTE: We have commented out unused columns from the source table
+      view.as(knex('licences').withSchema('water').select([
+        'licence_id AS id',
+        'region_id',
+        'licence_ref',
+        'is_water_undertaker',

```

Comment:
Same as previous comment

```suggestion
        'is_water_undertaker AS water_undertaker',

```